client/src/components/AlgSet/AlgSet.js

```diff
@@ -87,7 +87,7 @@ const AlgSet = ({ match }) => (
                   </ImportExportDialog>
                 </Fragment>
               )}
-              <CopyToClipboard text={window.location}>
+              <CopyToClipboard text={`${window.location.host}/alg-sets/${algSet.id}/${algSet.name}`}>
```

Comment:
We should actually ensure a nice formatting here, to make the name look like the example you pointed out. I think of wrapping `algSet.name` in `urlCase` function call. You can add that function to `logic/utils.js` and import here.
```js
export const urlCase = string =>
  string.replace(/\s+/g, '-').toLowerCase();
```